src/utils/variables.ts

```diff
@@ -24,7 +24,7 @@ const [gameContext, resetContext, defaultGameContext] = resettable(deepFreeze({
 //_______________________________audio, graphics________________________________
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
   audio: {
-    track: "",
+    track: '"*8"',
```

Comment:
I had to do this in order to trick the observer.
Since it seems to ignore when the gameContext.audio.track if it is already empty.
There is probably a better way but I'm not sure how to should be since all this observer logic is a bit out of my reach.

Reply:
i changed to undefined and it also works.